pages/common/locust.md

```diff
@@ -0,0 +1,27 @@
+# locust
+
+> Load-testing tool to determine number of concurrent users a system can handle.
+
+- Load-test "example.com" with web interface using locustfile.py:
+
+`locust --host={{http://example.com}}`
+
+- Use a different test file:
+
+`locust --locustfile={{test_file.py}} --host={{http://example.com}}`
+
+- Run test without web interface, spawning 1 user a second until there are 100 users:
+
+`locust --no-web --clients={{100}} --hatch-rate={{1}} --host={{http://example.com}}`
+
+- Start locust in master mode:
+
+`locust --master --host={{http://example.com}}`
+
+- Connect locust slave to master:
+
+`locust --slave --host={{http://example.com}}`
+
+- Connect locust slave to master on a different machine:
+
+`locust --slave --master-host={{master_ip}} --host={{http://example.com}}`
```

Comment:
Should we change it to "master_hostname" because the option mentions "master-host" ?

Reply:
I've only used IP for my tests but you can do both. I'll change it for simplicity. Thanks for the feedback.

Reply:
Done